constants/src/lib.rs

```diff
@@ -32,6 +32,12 @@ pub const MERKLE_ROOT_HISTORY_LENGTH: usize = 30;
 /// The fee that the protocol takes on each trade
 pub const PROTOCOL_FEE: f64 = 0.0006; // 6 bps
 
+/// The encryption key used by the protocol to collect fees
+///
+/// TODO: This is a dummy key, replace with real encryption key when it is created
+pub const PROTOCOL_ENCRYPTION_KEY: &str = 
+    "0x238605ef44e283ca92d1e2a373a8bb7885bddeff0be47d39d6016f270a9b900c79db2b3fe50110a1f9bcec1e924de82042be3351a0fb625d3f02042c28c3991e";
```

Comment:
fwiw, the encryption key that i've deployed the contracts with on the internal devnet is `0xc34790ba0d193ee43dc1e552f38f30b7db5321c1afc5f1749950f6191d1d9302122c27d82f28fe6b6b8afd4674fd97ff5c40f18acb8f5bcbe712dbb55cfc4918`, but that's only relevant for staging relayers